sdk/src/wallet/events/types.rs

```diff
@@ -24,7 +24,7 @@ pub struct Event {
     pub event: WalletEvent,
 }
 
-#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+#[derive(Clone, Debug, Eq, PartialEq)]
```

Comment:
putting this comment here ... but can you add a blank line after line 17? Why is the formatter not adding one? I though it would do that.

Reply:
Good catch, and no, formatter doesn't do that kind of changes unfortunately